synnax/pkg/api/api.go

```diff
@@ -355,10 +355,10 @@ func (a *API) BindTo(t Transport) {
 	t.ChannelRetrieveGroup.BindHandler(a.Channel.RetrieveGroup)
 
 	// FRAME
-	t.FrameWriter.BindHandler(a.Telem.Write)
-	t.FrameIterator.BindHandler(a.Telem.Iterate)
-	t.FrameStreamer.BindHandler(a.Telem.Stream)
-	t.FrameDelete.BindHandler(a.Telem.FrameDelete)
+	t.FrameWriter.BindHandler(a.Framer.Write)
+	t.FrameIterator.BindHandler(a.Framer.Iterate)
+	t.FrameStreamer.BindHandler(a.Framer.Stream)
+	t.FrameDelete.BindHandler(a.Framer.FrameDelete)
```

Comment:
You should change the handler to `Delete` instead of `FrameDelete`.

Reply:
Also, there's a bunch of variables and functions in `framer.go` that have names that start with `Frame` or `Framer` - those names should be changed.

Reply:
This file is in `package api` and not `package framer`, so a lot of these variables do need to be prefixed with `frame` since otherwise they collide with other names in the `api` package, like channel delete.